src/validations/trainingPreferences.ts

```diff
@@ -0,0 +1,41 @@
+import { NextFunction, Request, Response } from 'express';
+import Joi from 'joi';
+
+import { CustomError } from 'src/types/custom-error';
+
+export const allowedTrainingTypes = [
+  'STRENGTH',
+  'CARDIO',
+  'FUNCTIONAL',
+  'FLEXIBILITY',
+  'ENDURANCE',
+  'SPORTS_SPECIFIC',
+  'AGILITY',
+  'BODYWEIGHT',
+];
+
+export const trainingPreferenceSchema = Joi.object({
+  id: Joi.number().integer(),
```

Comment:
Remember to remove this id since we don't need to create the training preference